webserver/static/robots.txt

```diff
@@ -2,3 +2,5 @@ User-agent: *
 Disallow: /current-status
 Disallow: /user/
 Disallow: /1/
+Disallow: /api
```

Comment:
You are blocking URLs, not files or folders. It is better to have a trailing slash everywhere for consistency sake and to avoid blocking matching prefixes like `/apistuff`.